scipy/spatial/tests/test_distance.py

```diff
@@ -277,7 +280,7 @@ def _weight_checked(fn, n_args=2, default_axis=None, key=lambda x: x, weight_arg
                     ones_test=True, const_test=True, dup_test=True,
                     split_test=True, dud_test=True, ma_safe=False, ma_very_safe=False,
                     nan_safe=False, split_per=1.0, seed=0,
-                    compare_assert=partial(assert_allclose, atol=1e-5)):
+                    compare_assert=assert_allclose_forgiving):
```

Comment:
Was the linter complaining about this one? Callables as keyword defaults should be okay I'd think. 

Again, if it's a linter complaint I don't mind the change.

Reply:
Yes, here's the lint warning:

```
scipy/spatial/tests/test_distance.py:280:36: B008 Do not perform function call `partial` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable
```